src/openms/include/OpenMS/METADATA/ExperimentalDesign.h

```diff
@@ -34,70 +34,179 @@
 
 #pragma once
 
+#include <OpenMS/KERNEL/StandardTypes.h>
 #include <OpenMS/DATASTRUCTURES/String.h>
+#include <OpenMS/SYSTEM/File.h>
+#include <OpenMS/KERNEL/ConsensusMap.h>
+#include <OpenMS/KERNEL/FeatureMap.h>
+#include <OpenMS/METADATA/ProteinIdentification.h>
 
 #include <vector>
-#include <string>
 #include <map>
 #include <set>
 
 namespace OpenMS
 {
   /**
-  @brief A TSV and user friendly representation of the experimental design.
-  Used for loading and storing the experimental design in OpenMS.
+  @brief Representation of the Experimental Design in OpenMS. Instances can be loaded via
+   the ExperimentalDesignFile class.
 
-  @ingroup Metadata
+  @ingroup Format
   */
   class OPENMS_DLLAPI ExperimentalDesign
   {
   public:
-    /// 1) Mandatory section with run-level information of the experimental design.
-    ///    Required to process fractionated data and technical replicates.
-    ///
-    /// Run	Spectra File	Fraction	Technical Replicate   
-    ///   1	humanA.mzML	      1	              1          
-    ///   2	humanB.mzML	      1	              1          
-    ///   3	humanC.mzML	      2	              1              
-    ///   4	humanD.mzML	      2	              1
-    ///   5	humanE.mzML	      3	              1
-    ///   6	humanF.mzML	      3	              1          
-    ///   7	humanG.mzML	      3	              2       (<- example how a 2nd technical replicate is stored for fraction 3)
-    /// TODO: add possibility to provide optional columns that map to additional run-level specific meta data (see, for example, the mzTab specification)
-    class OPENMS_DLLAPI MSRun
+
+    class OPENMS_DLLAPI RunRow
```

Comment:
For multiplexed data one row does not necessarily correspond to an injection.
The name run section and sample section have been used to distinguish the
meta data associated with mapping quant. channels in fractionated files
and the sample specific meta data (like conditions etc.)

Reply:
we could also call this MSFileSection and MSFileRow? Would this be better?

Reply:
ok, in that case I think it should just be documented better. I dont really like the naming of "rows" and "columns" here since it referers to a specific file layout instead of concepts used by experimentalisits